src/games/strategy/triplea/ai/proAI/data/ProTerritory.java

```diff
@@ -432,4 +435,12 @@ public void setMaxScrambleUnits(final List<Unit> maxScrambleUnits) {
   public List<Unit> getMaxScrambleUnits() {
     return maxScrambleUnits;
   }
+
+  public List<Unit> getBombers() {
+    return bombers;
+  }
+
+  public void setBombers(final List<Unit> bombers) {
```

Comment:
Ah, so it does "getBombers().add()"

Would it not be cleaner to then have a `addBomber(Bomber bomber)` method?
Meanwhile, this set method is not used, and can be removed either way